Parsers/ASimAuthentication/Tests/PaloAlto_CortexDataLake_vimAuthentication_SchemaTest.csv

```diff
@@ -0,0 +1,79 @@
+Result
+"(1) Warning: Missing recommended field [DvcAction]"
+"(1) Warning: Missing recommended field [EventResultDetails]"
+"(1) Warning: Missing recommended field [TargetHostname]"
+"(2) Info: Missing optional alias [Application] aliasing non-existent column [TargetAppName]"
+"(2) Info: Missing optional field [ActingAppId]"
+"(2) Info: Missing optional field [ActingAppName]"
+"(2) Info: Missing optional field [ActingAppType]"
+"(2) Info: Missing optional field [ActorOriginalUserType]"
+"(2) Info: Missing optional field [ActorScopeId]"
+"(2) Info: Missing optional field [ActorScope]"
+"(2) Info: Missing optional field [ActorSessionId]"
+"(2) Info: Missing optional field [ActorUserId]"
+"(2) Info: Missing optional field [ActorUserType]"
+"(2) Info: Missing optional field [ActorUsername]"
+"(2) Info: Missing optional field [DvcDescription]"
+"(2) Info: Missing optional field [DvcInterface]"
+"(2) Info: Missing optional field [DvcMacAddr]"
+"(2) Info: Missing optional field [DvcOriginalAction]"
+"(2) Info: Missing optional field [DvcOsVersion]"
+"(2) Info: Missing optional field [DvcOs]"
+"(2) Info: Missing optional field [DvcScopeId]"
+"(2) Info: Missing optional field [DvcScope]"
+"(2) Info: Missing optional field [DvcZone]"
+"(2) Info: Missing optional field [EventOwner]"
+"(2) Info: Missing optional field [EventReportUrl]"
+"(2) Info: Missing optional field [EventSubType]"
+"(2) Info: Missing optional field [LogonMethod]"
```

Comment:
As per documentation here: we can evaluate LogonMethod using field: cn1Label
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/5c57fa97-7b5f-434b-bf09-32605f8bc396)

Reply:
https://docs.paloaltonetworks.com/cortex/cortex-data-lake/log-forwarding-schema-reference/network-logs/network-authentication-log

Reply:
TargetDvc details:
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/647566f6-280e-4014-8e51-3c4eec12017e)

Reply:
TargetDvcOS
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/5a82be28-99ba-447a-afbe-3e4588ef2a6b)

Reply:
LogonMethod is configured
TargetDvc set
TargetDvcOS not set. Doesn't seem like a good fit